shared/desktop/renderer/index.js

```diff
@@ -126,7 +126,12 @@ function setupApp (store) {
     _currentStore = subsetsRemotesCareAbout(store.getState())
 
     if (JSON.stringify(previousStore) !== JSON.stringify(_currentStore)) {
-      ipcRenderer.send('stateChange', store.getState())
+      ipcRenderer.send('stateChange', {
+        ...store.getState(),
```

Comment:
the way the flow works is
1. Main window contains the entire store
1. We figure out subsetsRemotesCareAbout to see if anyone cares about subsets changing.
1. We send the entire store to the node process * (see below)
1. The node process uses the selectors to send subparts to the registered remote components

The problem is we can't serialize immutable over the wire so our entire notifications substore is null. We used to use transit-immutable etc but i think i have some better ideas for how this can work which we can implement after mobile. for now i just override the remote notifications store as its only used by one remote window ever